Mytinerary/View Controllers/InputEventViewController.m

```diff
@@ -7,16 +7,53 @@
 //
 
 #import "InputEventViewController.h"
+#import "EventInputSharedView.h"
+#import "EventInputActivityView.h"
+
+static int const EVENTINPUTSHAREDVIEWHEIGHT = 600;
+static int const EVENTINPUTACTIVITYVIEWHEIGHT = 370;
```

Comment:
is this view controller hard coded to only work in portrait mode? If so, why?

Reply:
yes for now! planning to deal w/ autolayout and other responsive-ness later. would love your input on how to best go about that